cli/src/main/java/io/aklivity/zillabase/cli/internal/commands/start/ZillabaseStartCommand.java

```diff
@@ -2043,6 +1988,12 @@ CreateContainerCmd createContainer(
             DockerClient client)
         {
             ExposedPort exposedPort = ExposedPort.tcp(9092);
+            String volume = "/bitnami/kafka";
+
+            client.createVolumeCmd()
```

Comment:
How does this behave if the volume already exists?

Reply:
it was use existing volume instead of creating a new one.